tests/format/markdown/link/__snapshots__/format.test.js.snap

```diff
@@ -80,8 +80,8 @@ singleQuote: true
 ![link](https://www.google.fr/foo-<bar)
 ![link](https://www.google.fr/foo-%3Cbar) [link]:
 https://www.google.fr/()foo->bar [link]: https://www.google.fr/foo->bar [link]:
-https://www.google.fr/foo-%3Ebar [link]: https://www.google.fr/foo-<bar [link]:
-https://www.google.fr/foo-%3Cbar
+https://www.google.fr/foo-%3Ebar [link]: https://www.google.fr/foo-<bar
+[link]: https://www.google.fr/foo-%3Cbar
```

Comment:
This case is mistakenly parsed as HTML by `htmlFlowHackSyntax()`...
Probably we need to vendor html-text syntax if we don't accept 1) this change and 2) removal of spaces in quoted html attributes.